File: piedra_papel_tijera.js
var piedra = "piedra";
var papel = "papel";
var tijera = "tijera";
jugadas = [piedra, papel, tijera];

function juego(user, cpu) {
  if (user != cpu) {
    if (user == piedra) {
      res = "Ganaste!" ? cpu == tijera : "Perdiste!";
    } else if (user == papel) {
      res = "Ganaste!" ? cpu == piedra : "Perdiste!";
    } else if (user == tijera) {
      res = "Ganaste!" ? cpu == papel : "Perdiste!";
    } else {
      res = "Introduce una opción válida.";
    }
  } else {
    res = "Empate!";
  }
  return res;
}

index_jugadas = Math.floor(Math.random() * jugadas.length);
cpu = jugadas[index_jugadas];
user = prompt("Por favor introduce ");
console.log(`La compu jugó ${cpu}`);
console.log(juego(user, cpu));

File: piedra_papel_tijera.py
# Juego de piedra, papel o tijera hecho en python!

from random import choice

piedra = 'piedra'
papel = 'papel'
tijera = 'tijera'

jugadas = [piedra, papel, tijera]


def juego(user, cpu):
    if user != cpu:
        if user == piedra:
            res = 'Ganaste!' if cpu == tijera else 'Perdiste!'
        elif user == papel:
            res = 'Ganaste!' if cpu == piedra else 'Perdiste!'
        elif user == tijera:
            res = 'Ganaste!' if cpu == papel else 'Perdiste!'
        else:
            res = 'Please enter a valid option! '
    else:
        res = 'Empate!'
    return res


if __name__ == "__main__":
    user = input('Ingresa su jugada: ')
    cpu = choice(jugadas)
    print(f'La compu jugó {cpu}')
    print(juego(user, cpu))
